src/BloomExe/CollectionTab/CollectionTabView.Designer.cs

```diff
@@ -145,10 +145,10 @@ private void InitializeComponent()
             this._L10NSharpExtender.SetLocalizableToolTip(this._legacySettingsButton, null);
             this._L10NSharpExtender.SetLocalizationComment(this._legacySettingsButton, null);
             this._L10NSharpExtender.SetLocalizationPriority(this._legacySettingsButton, L10NSharp.LocalizationPriority.NotLocalizable);
```

Comment:
> L10NSharp.LocalizationPriority.NotLocalizable

Should be L10NSharp.LocalizationPriority.Medium
As it is, this is not internationalized.